packages/host/app/components/matrix/room.gts

```diff
@@ -32,9 +34,35 @@ interface RoomArgs {
     roomId: string;
   };
 }
+
+interface CommandArgs {
+  Args: {
+    command: any;
+  };
+}
+
+class CommandMessage extends Component<CommandArgs> {
```

Comment:
is this component really necessary?--it's just wrapping a single button. seems like we could just move this into the Room component instead